app/services/employee_finder.rb

```diff
@@ -0,0 +1,23 @@
+require "slack-ruby-client"
+
+class EmployeeFinder
+  def initialize
+    configure_slack
+  end
+
+  def employee_exists?(slack_username)
```

Comment:
so the way we've been writing these services classes, they all use the `run` method to process logic...thoughts on following that pattern here as well?